### PR TITLE
Created instructions component for home page

### PR DESCRIPTION
**Pull request by Kiran-Serolla**
# This is the instructions component for home page

Please provide a short summary explaining what this PR is about.

Fixes https://github.com/HackYourFuture-CPH/rate-my-cv/issues/101

# How to test?

This can be tested in storybook under InstructionsComponent

Screenshot (19)
# Checklist
- [ x] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [ x] I have made corresponding changes to the documentation
- [ x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Comment by h09shais** [comment]
[Build is failing!](https://github.com/HackYourFuture-CPH/rate-my-cv/pull/106/checks?check_run_id=1817607950)

File: src/client/components/InstructionsComponent/InstructionsComponent.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
+.container{
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    width: fit-content;
+
```
[the remark this answers is not in the document]
Comment (arya56) [reply src/client/components/InstructionsComponent/InstructionsComponent.css:6]: There need to be some white space above h2.. maybe with padding top.

File: src/client/components/InstructionsComponent/InstructionsComponent.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    margin-right: auto;
+    width: fit-content;
+    
+}
+.all-images{
+    display: flex;
+    margin-top: 30px;
+    padding: 50px;
+    
+
```
Comment (arya56) [inline src/client/components/InstructionsComponent/InstructionsComponent.css:13]: We might have some bottom padding to add more white spaces like the design we have in the figma.

**Review by pdsorensen — CHANGES_REQUESTED** [review]
Looks good. Check if any of the suggestions can be made and then add me as a reviewer. 

Also, be very careful of using generic words as css classes. A word like `image` could end up creating css conflicts which can be hard to debug.

File: src/client/components/InstructionsComponent/InstructionsComponent.css [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  width: 100px;
+  text-align: center;
+}
+
+.image-block img {
+  width: 100px;
+  padding: 15px;
+}
+
+h2 {
```
Comment (pdsorensen) [inline src/client/components/InstructionsComponent/InstructionsComponent.css:34]: No global styling please.

**Review by pdsorensen — CHANGES_REQUESTED** [review]

**Review by pdsorensen — APPROVED** [review]

**Pull request merged**